File: problems/daily/Java/1. Two Sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int[] ans = new int[2];
        int l = nums.length;
        for(int i = 0; i < l - 1; i++){
            int diff = target - nums[i];
            for(int k = i + 1; k < l; k++){
                if(diff == nums[k]){
                    ans[0] = i;
                    ans[1] = k;
                    break;
                }
            }
        }
        return ans;
    }

File: problems/daily/Python3/1. Two Sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        ans = [0,0]
        for i in range(len(nums)-1):
            for j in range(i+1,len(nums)):
                if target - nums[i] == nums[j]:
                    ans[0] = nums[i]
                    ans[1] = nums[j]
        return ans
    
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        for i in range(len(nums)-1):
            for j in range(i+1,len(nums)):
                if target - nums[i] == nums[j]:
                    return [i,j]